### PR TITLE
Improve radio checkboxes in the default editor theme

### PR DESCRIPTION
**Pull request by Calinou**
[The change was done for checkboxes already](https://github.com/godotengine/godot/pull/48954), but I forgot to do it for radio checkboxes in that PR.

## Preview

| Before | After |
|--------|-------|
|  |  |

| Before | After |
|--------|-------|
|  |  |

### CODE REVIEW

**Review by YeldhamDev — APPROVED** [review]

File: editor/icons/GuiRadioChecked.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg height="16" viewBox="0 0 16 15.999999" width="16" xmlns="http://www.w3.org/2000/svg"><g fill="#e0e0e0"><path d="m15 8a7 7 0 0 1 -7 7 7 7 0 0 1 -7-7 7 7 0 0 1 7-7 7 7 0 0 1 7 7" fill-opacity=".188235" stroke-width="2.333333"/><path d="m12 8a4 4 0 0 1 -4 4 4 4 0 0 1 -4-4 4 4 0 0 1 4-4 4 4 0 0 1 4 4" stroke-width="1.333333"/></g></svg>
+<svg height="16" viewBox="0 0 16 15.999999" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m15 8a7 7 0 0 1 -7 7 7 7 0 0 1 -7-7 7 7 0 0 1 7-7 7 7 0 0 1 7 7" fill="#699ce8" stroke-width="2.33333"/><path d="m12 8a4 4 0 0 1 -4 4 4 4 0 0 1 -4-4 4 4 0 0 1 4-4 4 4 0 0 1 4 4" fill="#fff" stroke-width="1.33333"/></svg>
```
Comment (akien-mga) [inline editor/icons/GuiRadioChecked.svg:1]: Missing `\n` at EOF.

**Comment by akien-mga** [comment]
Thanks!

**Comment by Calinou** [comment]
Since https://github.com/godotengine/godot/pull/48954 was cherry-picked for 3.4, this PR should also be cherry-picked for consistency :slightly_smiling_face:

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Pull request merged**